### PR TITLE
Some more rebranding

### PR DESCRIPTION
**Pull request by sandakersmann**
Replaced mentions of bitcoin with peercoin.

### CODE REVIEW

**Comment by sandakersmann** [comment]
@peerchemist @hrobeers Can you look at getting this merged before the release of v.0.6.0?

File: src/qt/bitcoinstrings.cpp [hunk 1]
```diff
@@ -162,7 +162,7 @@ QT_TRANSLATE_NOOP("bitcoin-core", "Rebuild block chain index from current blk000
 QT_TRANSLATE_NOOP("bitcoin-core", "Rescan the block chain for missing wallet transactions"),
 QT_TRANSLATE_NOOP("bitcoin-core", "Rescanning..."),
 QT_TRANSLATE_NOOP("bitcoin-core", "Run in the background as a daemon and accept commands"),
-QT_TRANSLATE_NOOP("bitcoin-core", "SSL options: (see the Bitcoin Wiki for SSL setup instructions)"),
```
Comment (hrobeers) [inline src/qt/bitcoinstrings.cpp]: Are you sure that wiki page exists for Peercoin? Maybe this should keep the referral to the Bitcoin Wiki?

Comment (sandakersmann) [reply src/qt/bitcoinstrings.cpp]: http://wiki.peercointalk.org/index.php?title=Main_Page

Comment (sandakersmann) [reply src/qt/bitcoinstrings.cpp]: Can we merge this first and then change it back?

Comment (hrobeers) [reply src/qt/bitcoinstrings.cpp]: why not just update the PR? you can just make the change and push it to your branch, the PR will update automatically.

Comment (sandakersmann) [reply src/qt/bitcoinstrings.cpp]: Ok. Done.

**Pull request merged**